contrib/chafa/patches/0001-disable-downloads-in-build-step.patch

```diff
@@ -0,0 +1,65 @@
+From a79063d2c767eaff3cc46da79e316957b69cb164 Mon Sep 17 00:00:00 2001
+From: ttyyls <contact@behri.org>
+Date: Thu, 29 Feb 2024 23:59:34 +0100
+Subject: [PATCH] disable downloads in build step
+
+---
+ configure.ac     | 24 ------------------------
+ docs/Makefile.am |  4 ++--
+ 2 files changed, 2 insertions(+), 26 deletions(-)
+
+diff --git a/configure.ac b/configure.ac
+index 1f037ab..3d62550 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -194,30 +194,6 @@ AS_IF([test "$enable_man" != no], [
+   ])
+ ])
+ 
+-AS_IF([ test "$enable_man" != no ], [
+-  dnl check for DocBook DTD in the local catalog
+-  JH_CHECK_XML_CATALOG([-//OASIS//DTD DocBook XML V4.1.2//EN],
+-     [DocBook XML DTD V4.1.2], [have_docbook_dtd=yes], [have_docbook_dtd=no])
+-  AS_IF([test "$have_docbook_dtd" != yes], [
+-    AS_IF([test "$enable_man" = yes ], [
+-      AC_MSG_ERROR([DocBook DTD is required for --enable-man])
+-    ])
+-    enable_man=no
+-  ])
+-])
+-
+-AS_IF([test "$enable_man" != no], [
+-  dnl check for DocBook XSL stylesheets in the local catalog
+-  JH_CHECK_XML_CATALOG([http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl],
+-     [DocBook XSL Stylesheets], [have_docbook_style=yes],[have_docbook_style=no])
+-  AS_IF([ test "$have_docbook_style" != yes ], [
+-    AS_IF([ test "$enable_man" = yes ], [
+-      AC_MSG_ERROR([DocBook XSL Stylesheets are required for --enable-man])
+-    ])
+-    enable_man=no
+-  ])
+-])
+-
+ AM_CONDITIONAL(ENABLE_MAN, test "$enable_man" != no)
+ 
+ AC_MSG_CHECKING([whether to generate man pages])
+diff --git a/docs/Makefile.am b/docs/Makefile.am
+index fa8ee8b..e07ba79 100644
+--- a/docs/Makefile.am
++++ b/docs/Makefile.am
+@@ -70,10 +70,10 @@ XSLTPROC_FLAGS = \
+         --stringparam html.stylesheet manpage.css
+ 
+ .xml.1:
+-	$(AM_V_GEN) $(XSLTPROC) $(XSLTPROC_FLAGS) http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl $<
++	$(AM_V_GEN) $(XSLTPROC) $(XSLTPROC_FLAGS) /usr/share/xsl-nons/docbook/manpages/docbook.xsl $<
```

Comment:
this doesn't do anything, xsltproc looks up the path in filesystem from the url

Reply:
the reason the configure check above fails is because it silently fails from missing `xmlcatalog`, which is in `libxml2-progs`

Reply:
maybe the -progs should be an xsl dependency already tbh